Add FORCE_BUILD option to cmake (#69)
eProsima/foonathan_memory_vendor: CMakeLists.txt

cmake_minimum_required(VERSION 3.14)
project(foonathan_memory_vendor VERSION "1.3.2")

> option(FORCE_BUILD "Force this package build even is already installed in the system" OFF)

Reviewer: ```suggestion
option(FOONATHAN_MEMORY_FORCE_VENDORED_BUILD "Force foonathan memory vendor package build even it is already installed in the system" OFF)
```

> if(FORCE_BUILD EQUAL OFF)

Reviewer: ```suggestion
if(NOT FOONATHAN_MEMORY_FORCE_VENDORED_BUILD)
```

Author: I committed all suggested changes

> option(FOONATHAN_MEMORY_FORCE_VENDORED_BUILD "Force foonathan memory vendor package build even it is already installed in the system" OFF)

Reviewer: ```suggestion
option(FOONATHAN_MEMORY_FORCE_VENDORED_BUILD "Force foonathan memory vendor package build even if it is already installed in the system" OFF)
```
